atomic_reactor/plugins/pre_download_remote_source.py

```diff
@@ -44,27 +44,6 @@ def __init__(self, tasker, workflow, remote_source_url=None,
         self.buildargs = remote_source_build_args or {}
         self.remote_source_conf_url = remote_source_configs
 
-    def get_remote_source_config(self, url, insecure=False):
-        """Get the configuration files associated with the remote sources
-
-        :param url: str, URL to cachito remote source configurations
-        :param insecure: bool, whether to verify SSL certificates
-
-        :return: list<dict>, configuration data for the given request.
-                 entries include path, type, and content
```

Comment:
Could we keep this part of the docstring somewhere? It would be nice to have the format of the response documented.

Reply:
Can we keep this function and just use session param to have better abstraction?

Reply:
Fixed.